<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Profilage</class>
 <widget class="QDialog" name="Profilage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxDetailEmploye">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>381</width>
     <height>431</height>
    </rect>
   </property>
   <property name="title">
    <string>Détails Employé</string>
   </property>
   <widget class="QLabel" name="labelMatricule">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Matricule </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNomEmploye">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPrenomEmploye">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Prénoms</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAgeEmploye">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Age</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSexeEmploye">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexe</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelPosteEmploye">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Poste</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserMatricule">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserNom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPrenom">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserAge">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserPoste">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowserSexe">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>280</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
